test/spec/modules/hybridBidAdapter_spec.js

```diff
@@ -0,0 +1,275 @@
+import { expect } from 'chai'
+import { spec } from 'modules/hybridBidAdapter'
```

Comment:
Modules defined locally should include the .js extension in the import path, so line 2 should be changed to: `import {spec} from 'modules/hybridBidAdapter.js`

Reply:
We fixed it